tests/__init__.py

```diff
@@ -0,0 +1,4 @@
+import os
+from pathlib import Path
+
+os.chdir(Path(__file__).parent.parent)
```

Comment:
What's the reason for this specific statement?

Reply:
Annoying test behavior when running individual tests, because of the plugin descovery logic (dynamic imports etc.)

Reply:
Fixing the plugin discovery instead would be better. It seems to me that the `import_statement` generated in e.g. `NormalizerResource` is indeed incorrect (full paths and package-like-statements mixed together).

I would suggest removing the `chdir` call since it's really not a good solution for import errors and setting your runner's working directory to the `../nl-kat-boefjes/` directory